src/widget/wfindonwebmenu.cpp

```diff
@@ -0,0 +1,282 @@
+#include "widget/wfindonwebmenu.h"
+
+#include <QDesktopServices>
+#include <QScreen>
+#include <QUrl>
+#include <QUrlQuery>
+#include <QtDebug>
+
+#include "track/track.h"
+#include "util/math.h"
+#include "util/qt.h"
+#include "util/widgethelper.h"
+
+namespace {
+
+QString composeActionText(const QString& prefix, const QString& trackProperty) {
+    return prefix + QStringLiteral(" | ") + trackProperty;
+}
+
+QString composeSearchQuery(const QString& trackAlbumOrTitle, const QString& artist) {
+    return trackAlbumOrTitle + QStringLiteral(" ") + artist;
+}
+
+QString composePrefixAction(WFindOnWebMenu::TrackSearchProperties trackSearchProperty) {
+    if (trackSearchProperty == WFindOnWebMenu::TrackSearchProperties::Title) {
+        return QObject::tr("Title");
+    } else if (trackSearchProperty == WFindOnWebMenu::TrackSearchProperties::Artist) {
+        return QObject::tr("Artist");
+    } else if (trackSearchProperty == WFindOnWebMenu::TrackSearchProperties::ArtistAndTitle) {
+        return QObject::tr("Title && Artist");
```

Comment:
The escaped & can be a translator trap.
How about  +?

Reply:
Oh okay, didn't notice that & would be an issue to translating. I will change it as + .